tokens/AERO-458bbf/info.json

```diff
@@ -1,13 +1,14 @@
-{ 
-"website": "https://aerovek.com",
-"description": "Aerovek is a social economic platform that aims to elevate the aviation industry using web3 technologies built on the Elrond Blockchain",  "ledgerSignature": "",
-"social": {
+{
+  "website": "https://aerovek.com",
+  "description": "Aerovek is a social economic platform that aims to elevate the aviation industry using web3 technologies built on the Elrond Blockchain",
```

Comment:
We should find & replace all Elrond occurrences.

Reply:
replaced the `Elrond network` \ `Elrond blockchain` by using MultiversX